demos/link-prediction/random-walks/cora-lp-demo.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>This is qualitatively quite different to how it was before. Maybe it's just because of the outlier at (2.4, 2) that's causing the main region to be much more compressed than it was? Thoughts?</p><p> </p><p>Also, it's not directly relevant to the changes in this PR, but this suffers from colourblindness problems like other plots we've talked about. Assuming the <code>alpha</code>  differences aren't critical, it could be switched to something like:</p><p> </p><pre class="ql-syntax" spellcheck="false">plt.scatter(X_transformed[:, 0], X_transformed[:, 1], c=link_labels_test, cmap="jet", alpha=0.8)
</pre><p>or for more direct control over the colours</p><p> </p><pre class="ql-syntax" spellcheck="false">plt.scatter(X_transformed[:, 0], X_transformed[:, 1], c=np.where(link_labels_test == 1, "g", "r"), alpha=0.8)
</pre><p>(this is reproducing the current numbers, but could be switched to <code>"b", "r"</code>  or something like that.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Seems like the scores are actually quite different from before too... I'll investigate a bit, but might also just be related to updating the dataset loading, and not necessarily the link prediction code</p><p> </p><p>There's a comment above the cell about which colour represents what, so I'll go with the direct control version with "b" and "r"</p><p> </p>

Reply:
Hmm actually I think I've done the evaluation incorrectly - there needs to be yet another `train_test_split` if I were to replicate existing code, I'll make an update